### PR TITLE
Update project to v1.8.0

### PR DESCRIPTION
**Pull request by SandyEma9**

### CODE REVIEW

**Comment by SandyEma9** [comment]
@gomboskriszta15 @robert-altom @RobertPoienar @ReichEdvin @iulianatodoran

**Comment by rucindrea** [comment]
LGTM~

File: tests/main_menu_page_test.py [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-    
+
     def setUp(self):
         self.main_menu_page = MainMenuPage(self.altdriver)
         self.main_menu_page.load()
 
     def test_main_menu_page_loaded_correctly(self):
-        assert(self.main_menu_page.is_displayed())
-
+        assert (self.main_menu_page.is_displayed())
```
Comment (robert-altom) [inline tests/main_menu_page_test.py]: Here you should remove the extra parentheses, leave just: 

```py
assert self.main_menu_page.is_displayed()
```

Please do this everywhere we use this pattern.

File: requirements.txt [hunk 2]
```diff
@@ -3,6 +3,6 @@ pytest==5.0
 pytest-selenium
 virtualenv
 Appium-Python-Client
-altunityrunner==1.5.7
+AltTester-Driver==1.8.0
 assertpy
 pytest-html
```
Comment (robert-altom) [inline requirements.txt]: We don't use `pytest-html`, `assertpy`, `pytest-selenium` or `selenium`, so you can remove them. 

Also there is no need to lock `pytest` to version 5.0, it's better to always install the latest version of `pytest`.

File: tests/game_play_test.py [hunk 3]
```diff
@@ -3,41 +3,42 @@
...
-        assert(self.main_menu_page.is_displayed())
+        assert (self.main_menu_page.is_displayed())
 
     def test_avoiding_obstacles(self):
         self.game_play_page.avoid_obstacles(5)
-        #check that player has 2 or 3 lives left (sometimes 1 life is lost right when stopping the tests)
-        assert_that(self.game_play_page.get_current_life()).described_as("player current life").is_greater_than(0) 
-
+    # check that player has 2 or 3 lives left (sometimes 1 life is lost right when stopping the tests)
+        assert_that(self.game_play_page.get_current_life()).described_as(
```
Comment (robert-altom) [inline tests/game_play_test.py]: Here you can just use assert like everywhere else: 

```python 
assert self.game_play_page.get_current_life() > 0
```

File: tests/game_play_test.py [hunk 4]
```diff
@@ -3,41 +3,42 @@
 from pages.game_play_page import GamePlayPage
 from pages.pause_overlay_page import PauseOverlayPage
 from pages.get_another_chance_page import GetAnotherChancePage
-import time
 from assertpy import assert_that
```
Comment (robert-altom) [inline tests/game_play_test.py]: Remove this too.

**Comment by robert-altom** [comment]
Also it would be nice to restructure the python examples, instead of having two different python packages for `pages` and `tests`, the `pages` package should be included in `tests`.

Instead of: 

```
tests/
   base_test.py
   ... 
pages/
  base_page.py
```

To: 

```
tests/
   pages/
      __init__.py
      base_page.py
      ...
   __init__.py 
   base_test.py
   ... 
```

This way we can remove all the `sys.path.append(os.path.dirname(__file__))` from the code.

**Comment by SandyEma9** [comment]
This is the structure in all example projects. I think it is better to keep package consistency

**Comment by robert-altom** [comment]
> This is the structure in all example projects. I think it is better to keep package consistency

I know but it's a hacky solution, that makes the code base difficult to change and maintain. 

I would be nice to restructure all python examples.

File: tests/game_play_test.py [hunk 5]
```diff
@@ -3,41 +3,40 @@
...
         self.pause_overlay_page = PauseOverlayPage(self.altdriver)
         self.get_another_chance_page = GetAnotherChancePage(self.altdriver)
 
     def tearDown(self):
         self.main_menu_page.load()
 
-
     def test_game_play_page_displayed_correctly(self):
-        assert(self.game_play_page.is_displayed())    
+        assert (self.game_play_page.is_displayed())
```
Comment (robert-altom) [inline tests/game_play_test.py]: You missed the extra parentheses here.

**Comment by SandyEma9** [comment]
@robert-altom I have resolved all comments.Please check when you have time

File: tests/base_test.py [hunk 6]
```diff
@@ -1,17 +1,14 @@
+from unittest import TestCase
+from alttester import AltDriver, AltPortForwarding
+from appium import webdriver
 import os
 import sys
 import time
 
 sys.path.append(os.path.dirname(__file__))
```
Comment (robert-altom) [inline tests/base_test.py]: It looks like you did not remove all calls to `sys.path.append(os.path.dirname(__file__))`.

**Pull request merged**